Makefile

```diff
@@ -5,7 +5,7 @@ GO_LICENSER_EXCLUDE=stacktrace/testdata
 check: precheck check-modules test
 
 .PHONY: precheck
-precheck: check-goimports check-lint check-vet check-dockerfile-testing check-licenses
+precheck: check-goimports check-lint check-vet check-dockerfile-testing check-licenses model/marshal_fastjson.go
```

Comment:
I think this would improve the UX when contributing, but I'm happy to remove this commit if someone disagrees.